packages/ddp-server/package.js

```diff
@@ -5,12 +5,13 @@ Package.describe({
 });
 
 Npm.depends({
-  "permessage-deflate": "0.1.7",
+  "permessage-deflate2": "0.1.8-alpha.1",
   sockjs: "0.3.24",
```

Comment:
How hard would it be to change from sockjs? Do we have testing covering the usage?

Reply:
It should involve some work, socket.io is a strong candidate, however I'm not sure there are direct tests for it